gst/nnstreamer/tensor_query/nnstreamer-edge-common.h

```diff
@@ -1,8 +1,8 @@
-/* SPDX-License-Identifier: LGPL-2.1-only */
+/* SPDX-License-Identifier: Apache-2.0 */
 /**
  * Copyright (C) 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
  *
- * @file   nnstreamer_edge_common.h
+ * @file   nnstreamer-edge-common.h
```

Comment:
Just curious. Is there any reason to change the file name?

Reply:
To sync with nnstreamer-edge repo.
refer: https://github.com/nnstreamer/nnstreamer-edge/blob/main/src/libnnstreamer-edge/nnstreamer-edge-common.c